src/Text/Layout/Table.hs

```diff
@@ -232,14 +232,17 @@ tableLines :: Cell a
            -> [RowGroup a] -- ^ Rows which form a cell together
            -> [String]
 tableLines specs TableStyle { .. } header rowGroups =
-    topLine : addHeaderLines (rowGroupLines ++ [bottomLine])
+    maybe id (:) topLine . addHeaderLines $ maybe id (\b -> (++[b])) bottomLine rowGroupLines
   where
     -- Helpers for horizontal lines that will put layout characters arround and
     -- in between a row of the pre-formatted grid.
 
-    -- | Generate columns filled with 'sym'.
-    fakeColumns sym
-                  = map (`replicateCharB` sym) colWidths
+    -- | Generate columns filled with 'sym', or blank spaces if 'sym' is of width 0.
```

Comment:
* If you refer to `sym` please make sure the parameter is actually named `sym`.
* English is not my mother tongue but I feel that comma does not belong there.

Reply:
Good catch. Fixed.

The comma can either be there or not depending on emphasis. Here the bit after the comma is secondary information, so separating it with a comma is appropriate.

Reply:
Thanks!